<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подбор рецептов по ингредиентам</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ingrFridgeTab">
       <property name="toolTip">
        <string/>
       </property>
       <attribute name="title">
        <string>Продукты в холодильнике</string>
       </attribute>
       <widget class="QTableWidget" name="ingrTabWdgt">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>741</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addIngrButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>460</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="listOfReceipts">
       <attribute name="title">
        <string>Список рецептов</string>
       </attribute>
       <widget class="QTableWidget" name="recpTblWdgt">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>741</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="showRecptBtn">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>470</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Посмотреть рецепт</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="availableReceipts">
       <attribute name="title">
        <string>Доступные блюда</string>
       </attribute>
       <widget class="QTableWidget" name="availableReceiptsTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>731</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="letsCookBtn">
        <property name="geometry">
         <rect>
          <x>274</x>
          <y>470</y>
          <width>81</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Приготовить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>470</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Количество порций</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>470</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
